util/src/dbutil.cpp

```diff
@@ -224,5 +281,60 @@ std::shared_ptr<odb::database> createDatabase(const std::string& connStr_)
   return db;
 }
 
+void createTables(
+  std::shared_ptr<odb::database> db_,
+  const std::string& sqlDir_)
+{
+  runSqlFiles(db_, sqlDir_,
+    [](const std::string& s_){
+      return removeByRegex(removeByRegex(removeByRegex(s_,
+        "CREATE UNIQUE INDEX", ";"),
+        "CREATE INDEX", ";"),
+        "ALTER TABLE", ";");
+    },
+    "Creating tables from file");
+}
+
+void createIndexes(
+  std::shared_ptr<odb::database> db_,
+  const std::string& sqlDir_)
+{
+  runSqlFiles(db_, sqlDir_,
+    [](const std::string& s_){
+      return removeByRegex(removeByRegex(s_,
+        "CREATE TABLE", ";"),
+        "DROP", ";");
+    },
+    "Creating indexes from file");
+}
+
+std::string updateConnectionString(
```

Comment:
We use this function only to update the database name. Why this function is so general.
- For example the key can be only a `host`, `database`, `user` or a `port`.  Why can I add new keys to the database connection string?
- What if the user calls this function with a value which contains a key value pair: `test; port=7777`.